etna/insights/migrations/0005_auto_20210715_1404.py

```diff
@@ -0,0 +1,20 @@
+# Generated by Django 3.1.8 on 2021-07-15 14:04
```

Comment:
Would you mind merging these migrations?

`python manage.py migrate insights 004`

Delete the two migrations…

Recreate
`python manage.py makemigrations`

Reply:
Done